vignettes/articles/validate-pr.Rmd

```diff
@@ -17,7 +17,7 @@ library(hubValidations)
 
 The `validate_pr()` functions is designed to be used to validate team submissions through Pull Requests on GitHub. 
 Only model output and model metadata files are individually validated using `validate_submission()` or `validate_model_metadata()` respectively on each file according to file type
-(_See the end of this article for details of the standard checks performed on each file. For more information on deploying optional or custom functions please check the article on [including custom functions](articles/custom-functions.html) (`vignette("custom-functions")`)_).
+(_See the end of this article for details of the standard checks performed on each file. For more information on deploying optional or custom functions please check the article on [including custom functions](deploying-custom-functions.html) (`vignette("articles/deploying-custom-functions")`)_).
```

Comment:
So THAT's why the `vignette()` commands weren't working! I didn't realize they needed to point to "articles/" first

Reply:
Well, I seem to remember (maybe I'm hallucinating though or maybe it just worked locally) them working before  (maybe before the dev version introductions, not 100% sure what broke the behaviour! ) This seems the only thing that fixes it.